### DISCUSSION TITLE
Weird interaction between show-icons and window height in percents

### DISCUSSION THREAD

**Discussion by elder-n00b** [davatorium/rofi#1587] [(Possible) Issues] [answered]
I have noticed this weird interaction between `configuration { show-icons: true; }` and `window { height: 80%; }`.

Context: I'd like to set the overall rofi window height, relative to the display size so that it looks right regardless I use the laptop screen or the external one. 
Of course I could achieve that by other means (perhaps using rofi's env var support), I just though I'd report the weirdness.

I have read https://github.com/davatorium/rofi/discussions/1407 and that lets me set the number of items displayed, using `listview { lines: ...; }`, but as I said I'd like to set the overall rofi window height instead, so I tried `window { height: 80%; }` and that has a very weird behavior: the 80% window flashes together with the blinking cursor (visually) behind the default height one, and when I type something, every key press results in the rofi window shrinking vertically some more (by the percent amount?), leaving previous "images" of it on screen. 
This only happens when both `show-icons` is true and `height` is in percents but not 100% (that works).
No theme specified, no other customization other than showing icons and window height.

I have no idea if this is a bug or I'm just doing it wrong, though I'd say it's likely both.

I searched around here and elsewhere, I could not find anything relevant.
Yes I read the pinned issue https://github.com/davatorium/rofi/issues/1397 too, I'm not migrating any theme from a previous version.
In fact, I'm new to using rofi, since yesterday.

**Note** that I'm using the `rofi-lbonn-wayland` fork built from the `1.7.3+wayland1-1-g9efa6504` tag, and running on Sway.
I'm asking here because there's no discussions there, and I'd like to know if this behavior is wayland specific or affects upstream rofi too.

**Comment by DaveDavenport** [comment, marked as answer]
works fine for me with default theme and Xorg.

**Comment by elder-n00b** [comment]
Interesting. That pretty much indicates it's a wayland thing.
Do you think I should report it to the lbonn fork repo?
P.S. should I mark the above message as answer, I think so?

**Reply by DaveDavenport** [reply]
this was my test line:
```bash
rofi -no-config -show-icons -show drun -theme-str 'window { height: 80%;}'
```

**Reply by elder-n00b** [reply]
Just tried it, glitch persists.
Should I mark your first reply as answer, should I open issue at lbonn fork?

**Reply by DaveDavenport** [reply]
yes.
